app/src/main/java/sdp/moneyrun/ui/game/GameLobbyActivity.java

```diff
@@ -141,6 +141,19 @@ public void onDataChange(@NonNull DataSnapshot snapshot) {
                         String newPlayersMissing = getString(R.string.lobby_player_missing,game.getMaxPlayerCount() - newPlayers.size());
 
                         playersMissing.setText(newPlayersMissing);
+                        GenericTypeIndicator<Boolean> coinIndicator = new GenericTypeIndicator<Boolean>() {
```

Comment:
Why is this variable called coinIndicator? I do not think it is an appropriate name as it has nothing to do with coins.

Reply:
I agree, will change this

Reply:
done